src/debugger/debugger.c

```diff
@@ -447,6 +453,11 @@ void xdebug_debugger_error_cb(zend_string *error_filename, int error_lineno, int
 		xdebug_hash_find(XG_DBG(context).exception_breakpoints, error_type_str, strlen(error_type_str), (void *) &extra_brk_info) ||
 		xdebug_hash_find(XG_DBG(context).exception_breakpoints, "*", 1, (void *) &extra_brk_info)
 	) {
+		if (stack_rebuilt) {
```

Comment:
If we are stopping on a pseudo exception we need to enable the debugger and rebuild the stack if needed. We use this boolean flag because the stack may have already been rebuilt if we have notified about this error